### PR TITLE
Remove the include/archive syntax as it's buggy

### PR DESCRIPTION
**Pull request by Crell**

### CODE REVIEW

File: src/configuration/yaml.md [hunk 1]
```diff
@@ -167,3 +141,18 @@ web:
                 '^/sitemap\.xml$':
                     allow: true
 ```
+
+
+### `!archive`
+
+Another custom tag available is `!archive`, which specifies a value is a reference to a directory on disk, relative to the location of the YAML file.  Essentially it defines the value of key as "this entire directory".  Consider this `services.yaml` fragment:
```
Comment (otaviojava) [inline src/configuration/yaml.md]: suggestion: "Essentially it defines the importance of key as "this entire directory".  Consider this `services.yaml` fragment:"

File: src/configuration/yaml.md [hunk 2]
```diff
@@ -167,3 +141,18 @@ web:
...
+
+```yaml
+mysearch:
+    type: solr:6.6
```
Comment (otaviojava) [inline src/configuration/yaml.md]: Maybe, update here to solr8 too

File: src/frameworks/drupal8/solr.md [hunk 3]
```diff
@@ -40,9 +40,7 @@ solrsearch:
     configuration:
         cores:
             maincore:
-                core_config: !include
-                    type: archive
-                    path: "solr-conf/6.x"
+                conf_dir: !archive "solr-conf/6.x"
```
Comment (otaviojava) [inline src/frameworks/drupal8/solr.md]: "solr-conf/8.x" ?

File: src/configuration/yaml.md (hunk 1 above)
Comment (Crell) [reply src/configuration/yaml.md]: Mm, that doesn't fit in English.  Translation issue I think.

File: src/frameworks/drupal8/solr.md (hunk 3 above)
Comment (Crell) [reply src/frameworks/drupal8/solr.md]: I'm not sure about changing the Drupal page yet, as there's some weirdness with the newer Solr bridge I've not sorted out yet.  See #1077.  I'd rather not mess with the version examples there until we know what else needs to change.

**Pull request merged**